atm.js

```diff
@@ -1,3 +1,185 @@
 $(document).ready(function(){
-// Uh oh -- it's saying `$` is undefined! Something's missing from `index.html`...
+
+var balance;
+var totalBalance;
+var userAmount;
+var nBalance;
+var userAmountN;
+var overDrawAmount;
+var checkingNewTotal;
+var savingsNewTotal;
+
```

Comment:
I would really avoid having this many global variables...it could mess up your code later when you don't know where scope is being applied.